_docs/cardano/2017-01-16-updating.md

```diff
@@ -235,8 +239,10 @@ The soft fork resolution rule works as follows:
   gets *adopted*. If more than one version has ≥ 75%, we take one of them
   deterministically.
 
-Note that new block versions can be adopted only at the beginning of an epoch,
-so all blocks in an epoch have the same block version.
```

Comment:
@ChShersh I think now I understand what was initial intent here. Most likely the idea was to say that **adopted block version** can't be changed during epoch (only between epochs), so all blocks in an epoch are verified according to the same rules (because rules are defined by **adopted** block version). Note that it differs from statement that all blocks have the same block version.